extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/CarMapSurfaceOwner.kt

```diff
@@ -118,55 +119,25 @@ internal class CarMapSurfaceOwner {
 
   fun scroll(distanceX: Float, distanceY: Float) {
     val carMapSurface = mapboxCarMapSurface ?: return
-    val handled = carMapObservers.any { it.onScroll(carMapSurface, distanceX, distanceY) }
-    if (handled) return
-
-    with(carMapSurface.mapSurface.getMapboxMap()) {
-      val fromCoordinate = visibleCenter
-      dragStart(fromCoordinate)
-      val toCoordinate = ScreenCoordinate(
-        fromCoordinate.x - distanceX,
-        fromCoordinate.y - distanceY
-      )
-      Logger.i(TAG, "scroll from $fromCoordinate to $toCoordinate")
-      setCamera(getDragCameraOptions(fromCoordinate, toCoordinate))
-      dragEnd()
+    val handled = gestureHandler.onScroll(carMapSurface, visibleCenter, distanceX, distanceY)
+    if (!handled) {
+      defaultGestureHandler.onScroll(carMapSurface, visibleCenter, distanceX, distanceY)
```

Comment:
since we fallback to default gesture handler anyway, shall we consider making the `DefaultGestureHandler` an internal class?

Reply:
Yeah. I don't like the multiple instances of gesture handler inside.

Like you're saying below, we can still have the open class, and also make the gesture handler nullable

Reply:
But the, return "handled boolean" also has no use.

Reply:
Resolving as it is addressed